Testreglar/4.1.2/Nett/nett-4.1.2c.json

```diff
@@ -0,0 +1,133 @@
+{
+	"namn": "Nett-4.1.2c Iframe har et tilgjengelig navn som beskriver formålet med innholdet i iframe 2023",
+	"id": "nett-4.1.2c",
+	"testlabId": 437,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Der det er brukt iframe-elementet, har iframe et tilgjengelig navn, som beskriver formålet med innholdet i iframe.</p>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsiden &lt;iframe&gt;-element?",
+			"ht": "<p>Søk etter <code>&lt;iframe&gt;</code>-elementet i kildekoden til testsiden. </p>\n<p><strong>Merk:</strong> Du skal vurdere kun iframe som er visuell synlige og inkludert i accessibilty tree. Skjult iframe for eksempel fra Google Analytics er ikke omfattet av kravet, fordi det ikke er en brukergrensesnittkomponent.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke iframe."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken iframe tester du?",
+			"ht": "<p>Beskriv <code>&lt;iframe&gt;</code>-elementet, slik at det er mulig å identifisere det i ettertid. Hvis det gjelder flere elementer, registrerer du et og et.</p>",
+			"type": "tekst",
+			"label": "Iframe element:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Har &lt;iframe&gt;-elementet et tilgjengelig navn?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Inspiser <code>&lt;iframe&gt;</code>-elementet i Chrome</li>\n<li>Velg <code>&lt;iframe&gt;</code>-elementet i kildekoden</li>\n<li>Bruk Accessibility Checker verktøy i inspiser</li>\n<li>Under Computed Properties, finner du blant annet \"Name\" og \"Role\"</li>\n<li>Sjekk om informasjon under \"Name\" ikke er tomt</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Iframe har ikke et tilgjengelig navn."
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				}
+			},
+			"kilde": [
+				"ARIA14",
+				"ARIA16",
+				"H64"
+			]
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Hvilket attributt gir tilgjengelig navn til iframe?",
+			"ht": "<ul>\n<li>Sjekk dette under \"Name\" under Computed Properties i Accessibility Checker</li>\n<li>Velg et alternativ.</li>\n</ul>",
+			"type": "radio",
+			"svarArray": [
+				"aria-labelledby",
+				"aria-label",
```

Comment:
Hva skal vi bruke denne informasjonen til?

Reply:
Bare å ha referanse om hvilket attributt gir tilgjengelige navn til iframe, slik at den kan gjenkjenne om det er behov for det.